Classes/WPEditorViewController.m

```diff
@@ -1062,18 +1062,19 @@ - (void)showInsertImageAlternatePicker
 
 - (void)recoverFromViewSizeChange
 {
+    // This hack forces the input accessory view to refresh itself and resize properly.
     if (self.isFirstSetupComplete) {
-        // Important: This is a complete and utter hack that compensates for the input accessory view
-        // not properly changing size classes (resizing) when the rest of the views in the editor VC do.
-        // Toggling the HTML button on the input bar quickly does not affect the view and forces the
-        // input accessory view (the format bar) to update itself. FWIW, setNeedsDisplay and
-        // setNeedsLayout do NOT work.
         if ([self.editorView isInVisualMode]) {
-            [self.editorView showHTMLSource];
-            [self.editorView showVisualEditor];
+            [[self.editorView focusedField] blur];
+            [[self.editorView focusedField] focus];
```

Comment:
Good add @SergioEstevao! Works great on the iPad Pro with multitasking and rotation.